src/components/Settings/components/Pages/FeedbackSetting.jsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
 
 const FeedbackSetting = () => {
-	return <div>This is the FEEDBACK settings page</div>;
+
+	return (
+		<div>
+			<div className="h-[calc(100vh-5em)]">
+				<iframe
+					className="clickup-embed clickup-dynamic-height"
+					src="https://forms.clickup.com/24323508/f/q69dm-29381/B9D37296WIKIQ99ME2"
+					width="100%"
+					height="100%"
+				></iframe>
+				<script async src="https://app-cdn.clickup.com/assets/js/forms-embed/v1.js"></script>
```

Comment:
Is het niet beter om een extern script via de useEffect in te laden? 

```
useEffect(() => {
  const script = document.createElement('script');

  script.src = "https://use.typekit.net/foobar.js";
  script.async = true;

  document.body.appendChild(script);

  return () => {
    document.body.removeChild(script);
  }
}, []);
```

Reply:
Kan je uitleggen waarom dit beter zou zijn? Ik volg je hier wel in.

Reply:
We kunnen zo vermijden dat het script meerdere malen zal worden ingeladen bij een rerender.

Reply:
Awel maar dat is deel van die Clickup embedded code, dus I don't know 😅

Reply:
> We kunnen zo vermijden dat het script meerdere malen zal worden ingeladen.

Idd, klopt! We zouden hier een custom hook voor kunnen schrijven 😉